src/CosmosDBStudio.Model/Services/Implementation/AccountDirectory.cs

```diff
@@ -27,7 +27,7 @@ public IEnumerable<object> GetChildNodes(string folderPrefix)
             var accounts = new List<CosmosAccount>();
             var prefix = folderPrefix.TrimStart('/');
 
-            foreach (var (_, account) in _accounts)
+            foreach (var (_, account) in _accounts.OrderBy(a => a.Value.Id))
```

Comment:
> I think this should be done in the ViewModel layer, where GetRootNodes or GetChildNodes are called.
> 
> Also, I think it's not quite right, for two reasons:
> 
> * the account id is a GUID; sorting by a GUID doesn't help the user, you want to sort by name
> * it doesn't take folders into account
> 
> I think it'll be easier to sort on the result of GetRootNodes/GetChildNodes. I realize it's a bit ugly, since it returns a collection of objects... You'll need to cast to either CosmosAccount or CosmosAccountFolder

On it.